src/settings.plugin.js

```diff
@@ -115,56 +112,58 @@ wikiaJWPlayerSettingsPlugin.prototype.addSettingsContent = function (div) {
 		div.appendChild(this.qualityLevelsList);
 	}
 
+	if (this.config.showCaptions) {
+		this.createCaptionsList();
+		div.appendChild(this.captionsList);
+	}
+
 	return div;
 };
 
 wikiaJWPlayerSettingsPlugin.prototype.createSettingsListElement = function () {
 	var settingsList = document.createElement('ul');
 
-	settingsList.classList.add('wikia-jw-settings__list');
-	settingsList.classList.add('wds-list');
+	settingsList.className = 'wikia-jw-settings__list wds-list';
 
 	if (this.config.showQuality) {
 		settingsList.appendChild(this.createQualityButton());
 	}
 
+	if (this.config.showCaptions) {
+		settingsList.appendChild(this.createCaptionsButton());
+	}
+
 	if (this.config.showAutoplayToggle) {
 		settingsList.appendChild(this.createAutoplayToggle());
 		this.show();
 	}
 
-	if (this.config.showCaptionsToggle) {
-		settingsList.appendChild(this.createCaptionsButton());
-		this.addCaptionListener();
-	}
-
 	return settingsList;
 };
 
-wikiaJWPlayerSettingsPlugin.prototype.createSVG = function (svgHtml) {
-	return domParser.parseFromString(svgHtml, 'image/svg+xml').documentElement;
-};
+wikiaJWPlayerSettingsPlugin.prototype.createSubmenuWrapper = function () {
+	var backButton = document.createElement('li'),
```

Comment:
So, is it a button or a list element? ;>

Reply:
a list element that acts like button 👌